service/gcs/runtime/runc/utils.go

```diff
@@ -62,8 +62,8 @@ func (r *runcRuntime) makeContainerDir(id string) error {
 }
 
 // getLogPath returns the path to the log file used by the runC wrapper.
-func (r *runcRuntime) getLogPath() string {
-	return filepath.Join(containerFilesDir, "log.log")
+func (r *runcRuntime) getLogPath(id string) string {
+	return filepath.Join("/tmp/gcs", id, "runc.log")
```

Comment:
nit: It would be nice to have "/tmp/gcs" in some constant somewhere, either by having an actual constant or using `getStorageRootPath()`